packages/web-wallet/pages/credentials/show/index.tsx

```diff
@@ -57,16 +59,20 @@ type DocumentItem = {
 
 const truncationLength: number = process.env.NEXT_PUBLIC_TRUNCATION_LENGTH ? Number(process.env.NEXT_PUBLIC_TRUNCATION_LENGTH) : 8
 
-const ShowCredentialDetails = () => {
+type Props = {
+  credentialRole: CredentialRole
+}
+
+const ShowCredentialDetails: FC<Props> = (props: Props): ReactElement => {
+  const {credentialRole} = props
   const translate = useTranslate()
-  const {id} = useParams()
+  const params = useParams()
+  const {id} = params
   const [credentialSummary, setCredentialSummary] = useState<CredentialSummary | undefined>(undefined)
-
-  const credentialResult = useOne<Credential, HttpError>({
-    dataProviderName: 'supaBase',
-    resource: 'credential',
+  const credentialResult = useOne<DigitalCredential, HttpError>({
+    resource: DataResource.CREDENTIALS,
     id,
-    meta: {idColumnName: 'hash'},
+    meta: {variables: {credentialRole: credentialRole}},
```

Comment:
Why is there no idColumnName anymore? Isn't that needed?

Reply:
The dataProvider is replaced with credentialDataProvider.ts using the agent backend.
agent.crsGetCredentials doesn't do anything with that field so it was removed.